Automattic/wp-calypso
# Get Apps: simplify the code to request sending a SMS

There is code that sends a POST REST request to initiate sending a SMS message to the user. It displays a success or error notice depending on the result.

Such a feature doesn't need three Redux actions, reducer, selectors and a separate data layer module

It can be inlined into the component in approx 20 lines of code and still use the full data layer. 

I noticed this when reviewing #32034 (migrating Get Apps CSS)

**How to test:**
Go to the `/me/get-apps` page, enter phone number, click "Text me a link". Verify that the REST request was sent and success/error notice was displayed.
